app/services/workflow/instance/process/complete.rb

```diff
@@ -0,0 +1,48 @@
+module Workflow
+  class Instance::Process
+    class Complete < BaseService
+      # track startability of new processs as a metric, e.g. what did we unlock and as of when
+      # that way we can see what's waiting in someone's court.
+      # particularly for non-workers
+
+      # after we complete, we need to see what's unlocked.
+      # that's done later since state is changed..
+      def initialize(process)
+        @process = process
+      end
+
+      def run
+        # @process.completed = true # do processes have these fields?  separate this to another table?
+        # @process.completed_at = DateTime.now # this is on the step.
+        @process.status = V1::Statusable::DONE
```

Comment:
🤔 I don't see a `status` column on `process`. Only `completion_status`, which is an enum.